apps/crunchy-postgres/workload/crunchy_postgres_loadgen.yml

```diff
@@ -3,11 +3,12 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: crunchy-loadgen
-  namespace: litmus
 spec:
   template:
     metadata:
       name: crunchy-loadgen
+      labels:
+        loadgen_lkey: loadgen_lvalue
       
     spec:
```

Comment:
The image openebs/postgres-client is not there in docker hub. Can we make sure the image is pushed first before going ahead with this PR?

Reply:
@dargasudarshan  Our travis build automatically pushes images for ansible-runner with ci tag for every commit.

Reply:
@shashank855 Here i am not talking about ansible-runner image, it is the openebs/postgres-client image which has to pushed.

Reply:
@shashank855 can you change the image name to `openebs/tests-postgresql-client`